controller/src/main/java/io/pravega/controller/task/KeyValueTable/TableMetadataTasks.java

```diff
@@ -147,6 +147,7 @@ public void initializeStreamWriters(final EventStreamClientFactory clientFactory
                                                    .thenCompose(x -> isCreateProcessed(scope, kvtName, kvtConfig,
                                                            createTimestamp, executor, context));
                                        }
+                                       log.info("KeyValue table {}/{} already exists", scope, kvtName);
```

Comment:
Add requestId

Reply:
Added request id in info log message.

Reply:
System test runs were successful: **pravega-branch-builder/1788/**